config.cson

```diff
@@ -1857,6 +1862,11 @@ fileIcons:
 		match: ".dig"
 		colour: "medium-orange"
 		uses: "YAML"
+		
+	"Digital Command Language":
+        icon: "openvms"
```

Comment:
This line is indented using spaces. Please use tabs.

**NB:** You should install the [`editorconfig`](https://atom.io/packages/editorconfig) package, which will help to retain a project's coding-style automatically.